MAINTAINERS

```diff
@@ -50,10 +50,9 @@ maintainer candidates are selected and proposed on the maintainers mailing list.
 
 After a candidate has been announced on the maintainers mailing list, the
 existing maintainers are given five business days to discuss the candidate,
-raise objections and cast their vote. Candidates must be approved by the BDFL
-and at least 66% of the current maintainers by adding their vote on the mailing
+raise objections and cast their vote. Candidates must be approved by at least 66% of the current maintainers by adding their vote on the mailing
```

Comment:
nit: 80 columns rule? :innocent:

Reply:
![nope](http://gifs.joelglovier.com/no/nope-duck.gif)

Reply:
OH man.. looks like the TSC will have our first dispute issue to resolve sooner than I thought! :joy:

Reply:
I just won't be the one to merge this one! :laughing: